ui/src/actions/actions-ui.js

```diff
@@ -0,0 +1,18 @@
+import { REVERSE_SORT } from '../actions/types.js';
+import { SortByDirection } from '@patternfly/react-table';
+
+export function reverseAppsTableSort (direction, index, apps) {
```

Comment:
This can be made into an arrow function and can be changed to a const to make it immutable (which is what we want as we are not modifying existing states, we are creating a new one each time.

```suggestion
export const reverseAppsTableSort = (direction, index, apps) => {
```